### PR TITLE
Fixing RBAC test case to incorporate changes done as part of SRVKP-2256

### PR DESCRIPTION
**Pull request by VeereshAradhya**
Fixing RBAC test case to incorporate changes done as part of SRVKP-2256

### CODE REVIEW

File: specs/operator/rbac.spec [hunk 1]
```diff
@@ -14,6 +14,8 @@ Importance: High
 This scenario helps you to disable creation of RBAC resources at cluster level.
 
 Steps:
+  * Update TektonConfig CR to use param with name createRbacResource and value "true" to "enable" auto creation of RBAC resources
```
Comment (savitaashture) [inline specs/operator/rbac.spec:17]: By default `createRbacResource` is enbled so do we need to add this step :thinking:

Comment (pradeepitm12) [reply specs/operator/rbac.spec:17]: If `createRbacResource` is `enabled` by default , we don't need this.

Comment (savitaashture) [reply specs/operator/rbac.spec:17]: Yup we can remove then

Comment (VeereshAradhya) [reply specs/operator/rbac.spec:17]: If I use a cluster on which the `createRbacResource` is set to `false`, the test fails, so I am just setting this as a pre condition

**Review by savitaashture — APPROVED** [review]
:tada:

**Pull request merged**